docs/current_docs/integrations/snippets/openshift.yaml

```diff
@@ -7,12 +7,12 @@ engine:
     tag: v0.10.2
```

Comment:
If this is the Dagger version, should we bump it to the recently released `v0.11.0`?

Reply:
Added as suggestion